.travis.yml

```diff
@@ -21,6 +21,14 @@ env:
     - RAILS_ENV=test
     - CC_TEST_REPORTER_ID=f39e060a8b1a558ebd8aff75d5b9760bf1ae98f3f85d628ae28814f3c66438cd
     - DATABASE_URL=postgres://postgres@localhost/
+    - APP_PROTOCOL=http://
```

Comment:
@mstruve I think the amount of dummy env variables needed to get the Rails runner to start up is manageable.